core/src/main/java/io/questdb/griffin/SqlCompiler.java

```diff
@@ -1218,7 +1219,7 @@ private CompiledQuery alterTableAddColumn(int tableNamePosition, String tableNam
 
                 tok = SqlUtil.fetchNext(lexer);
                 if (tok != null && tok.charAt(0) != ')') {
-                    int geosizeBits = SqlParser.parseGeoHashBits(lexer.lastTokenPosition(), 0, tok);
+                    int geosizeBits = GeoHashUtil.parseGeoHashBits(lexer.lastTokenPosition(), 0, tok);
```

Comment:
we also have class `GeoHashes`, which is a similar collection of static methods for GeoHash handling

Reply:
Yes, I saw that one. I decided to introduce `GeoHashUtil` since `GeoHashes` is located in `*.cairo` and not aware of SQL classes. `GeoHashUtil`, on the other hand, is in `*.griffin` package and contains utility methods that, for instance, throw `SqlException`.

Do you think I should merge them together?

Reply:
nope, all good